addons/website_event_track/controllers/event_track.py

```diff
@@ -319,34 +319,33 @@ def _split_track_by_days(self, track, local_tz):
         Also return a set of all the time slots
         """
         start_date = fields.Datetime.from_string(track.date).replace(tzinfo=pytz.utc).astimezone(local_tz)
-        start_datetime = self.time_slot_rounder(start_date, 15)
-        end_datetime = self.time_slot_rounder(start_datetime + timedelta(hours=(track.duration or 0.25)), 15)
-        time_slots_count = int(((end_datetime - start_datetime).total_seconds() / 3600) * 4)
+        track_start_time_slot = start_time_slot = self.time_slot_rounder(start_date, 15)
+        end_datetime = self.time_slot_rounder(start_time_slot + timedelta(hours=(track.duration or 0.25)), 15)
```

Comment:
😴-coder proofing while I'm in the 😴-zone

Let's try keeping the names different enough, and I'd shorten "time_slot" to "slot" since in this context it's implicit
```suggestion
        track_start = current_slot_start = self.time_slot_rounder(start_date, 15)
        current_slot_end = self.time_slot_rounder(current_slot_start + timedelta(hours=(track.duration or 0.25)), 15)
```